libafl_frida/src/lib.rs

```diff
@@ -42,7 +42,7 @@ It can report coverage and, on supported architecutres, even reports memory acce
     deny(
         bad_style,
         const_err,
-        dead_code,
+        // dead_code,
```

Comment:
Do we really need this?

Reply:
that was for debugging
sorry, I forgot to change it back